### ISSUE TITLE
Commit hook doesn't work when Python is Python2

### ISSUE THREAD

**Issue by mkesper** [AleksaC/hadolint-py#1]
```Python traceback
$ git commit --amend
[INFO] Installing environment for https://github.com/AleksaC/hadolint-py.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/python', '-mvirtualenv', '/home/user/.cache/pre-commit/repo0aHJeO/py_env-python2.7', '-p', '/usr/bin/python')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/runpy.py", line 163, in _run_module_as_main
        mod_name, _Error)
      File "/usr/lib/python2.7/runpy.py", line 111, in _get_module_details
        __import__(mod_name)  # Do not catch exceptions initializing package
      File "/home/user/.local/lib/python2.7/site-packages/virtualenv/__init__.py", line 3, in <module>
        from .run import cli_run, session_via_cli
      File "/home/user/.local/lib/python2.7/site-packages/virtualenv/run/__init__.py", line 13, in <module>
        from .plugin.activators import ActivationSelector
      File "/home/user/.local/lib/python2.7/site-packages/virtualenv/run/plugin/activators.py", line 6, in <module>
        from .base import ComponentBuilder
      File "/home/user/.local/lib/python2.7/site-packages/virtualenv/run/plugin/base.py", line 5, in <module>
        from backports.entry_points_selectable import entry_points
    ImportError: No module named entry_points_selectable
```

**Comment by mkesper** [comment]
On another system where /usr/bin/python is python3, everything works as expected.

**Comment by AleksaC** [comment]
Sorry for responding this late, the issue somehow slipped past me. Python 2 hasn't been supported by this hook (`python_requires = >=3.6` in `setup.cfg`) and considering that its been almost 2 years since its EOL it never will be.

**Comment by mkesper** [comment]
Thanks a lot, that should solve it.
